lib/pretty-print.js

```diff
@@ -146,7 +183,9 @@ function drawBranch (pathSeg, prefix, endBranch, noPrefix, rootBranch) {
   return branch
 }
 
-function prettyPrintFlattenedNode (flattenedNode, prefix, tail) {
+function prettyPrintFlattenedNode (flattenedNode, prefix, tail, opts, _fmw) {
```

Comment:
The `_fmw` is required and we should move the function into the Router prototype or use `function.call(context)`

```suggestion
function prettyPrintFlattenedNode (flattenedNode, prefix, tail, opts) {
```